packages/api/src/util/logger/logger.test.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-ignore
```

Comment:
This is the only file relevant to this PR for now (the rest is because this PR is based on #115).